content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -0,0 +1,656 @@
+
+1. **No dynamic switching to ambient mode:** Ambient mode can only be enabled on a new Istio mesh control plane that is deployed using ambient profile or helm configuration. An existing Istio mesh deployed using a pre-ambient profile for instance can not be dynamically switched to also enable ambient mode operation.  
+
+1. **Restrictions with Istio PeerAuthentication:**  as of the time of writing, the PeerAuthentication resource  is not supported by all components (i.e. Waypoint proxies) in Istio ambient mode. Hence it is recommended to only use the STRICT mTLS mode currently, or simply use default Istio settings which automatically enable mTLS on all endpoints. Like many of the other alpha stage caveats, this shall be addressed as the feature moves toward beta status. 
+
+1. **Istioctl CLI gaps:** There may be some minor functional gaps in areas such as istioctl CLI output displays when it comes to displaying or monitoring Istio's ambient mode related information. These will be addressed as the feature matures.
+
+### Environment used for this guide 
+
+For the examples in this guide, we used a deployment of Istio version 1.19.0 on a `kind` cluster of version 0.20.0 running Kubernetes version 1.27.3. However these should also work on any Kubernetes cluster at version 1.24.0 or later and Istio version 1.18.0 or later.  It would be recommended to have a cluster with more than 1 worker node in order to fully exercise the examples described in this guide. Refer to the [installation user guide](/docs/ops/ambient/usage/install/) or [Getting started guide](/docs/ops/ambient/getting-started/) information on installing Istio in ambient mode on a Kubernetes cluster. 
+
+## Functional Overview {#functionaloverview}
+
+In this section we provide a functional summary of the control plane and data plane behavior of current version of the ztunnel proxy. Detailed description of architecture internals of the ztunnel proxy is out of scope for this guide.  
+
+### Control plane overview
+
+The figure shows an overview of the control plane related components and flows between ztunnel proxy and the `istiod` control plane..
+
+{{< image width="100%"
+    link="ztunnel-architecture.png"
+    caption="Ztunnel architecture"
+    >}}
+
+
+Each instance of the ztunnel proxy uses the Envoy xDS apis to receive certificates, discovery and configuration information from the istio control plane (`istiod`) on behalf of all pods and endpoints associated with it. In particular the ztunnel proxy obtains mTLS certificates for all Service accounts of all pods that are associated with it. Since a single ztunnel proxy performs both the data plane and the control plane operations across multiple service accounts, it is a multi-tenant component of the mesh infrastructure in contrast with Istio side car proxies that handle control plane and data plane operations on a per application endpoint or pod basis.
```

Comment:
> I think it's a bit confusing to say that ztunnel performs "control plane operations". I think you mean that it handles the tasks of communicating with the control plane.

changed it to ..
Since a single ztunnel proxy is shared by pods that use multiple different service accounts on a node, it is a multi-tenant component of the mesh infrastructure in contrast with Istio side car proxies that handle control plane and data plane operations on a per application endpoint or pod basis.

Reply:
Okay, used the suggested text.